src/main/java/duke/Save.java

```diff
@@ -0,0 +1,89 @@
+package duke;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.FileNotFoundException;
+import java.security.KeyStore;
+import java.util.Scanner;
+
+public class Save {
+    public static final int LENGTH_MARKER = 4;
+    public static String filePath = "data.txt";
+    public static String originalInputPath = "inputFile.txt";
+    public static TaskManager taskList = new TaskManager();
+    // load data to this taskList and pass the reference of the taskList to the one in Duke.java
```

Comment:
This comment seems to be misplaced as two lines after there is a method `saveData` which seems to be different from loading data.